p63eec/src/components/header.jsx

```diff
@@ -0,0 +1,31 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
+import React from 'react';
+import Container from 'react-bootstrap/Container'
+import Row from 'react-bootstrap/Row'
+import Col from 'react-bootstrap/Col'
+
+function Header () {
+    return(
+        <>
+        <Container style={{color:"white", backgroundColor: "#bf2886"}}>
```

Comment:
use classes instead of inline stlye

```
 <Container className="header">
```

and then in the css file (App.css)

```
.header {
  color: white;
  background-color: #bf2886;
}
```